src/commands/tokens/send_ft/amount_ft.rs

```diff
@@ -61,14 +57,6 @@ impl AmountFtContext {
             ft_contract_account_id: previous_context.ft_contract_account_id,
             receiver_account_id: previous_context.receiver_account_id,
             ft_transfer_amount,
-            memo: scope.memo.as_ref().and_then(|s| {
```

Comment:
I would prefer this to be honest, over custom type that is not needed yet in the end.

Reply:
Done